artemis/modules/webapp_identifier.py

```diff
@@ -38,6 +38,9 @@ def _identify(url: str) -> WebApplication:
             if re.search(webapp_sig, response.text):
                 return webapp_id
 
+        if "/wp-includes/css/" in response:
```

Comment:
is it not better to check for tags (`<script>` I assume), that have `.*/wp-includes/.*` in their attributes?

Reply:
that would require HTML parsing